diskqueue.go

```diff
@@ -531,11 +754,11 @@ func (d *diskQueue) retrieveMetaData() error {
 	defer f.Close()
 
 	// if user is using disk space limit feature
-	if d.diskLimitFeatIsOn {
-		_, err = fmt.Fscanf(f, "%d\n%d,%d,%d\n%d,%d,%d,%d\n",
+	if d.enableDiskLimitation {
+		_, err = fmt.Fscanf(f, "%d\n%d,%d,%d\n%d,%d,%d\n",
 			&d.depth,
 			&d.readFileNum, &d.readMessages, &d.readPos,
-			&d.writeBytes, &d.writeFileNum, &d.writeMessages, &d.writePos)
+			&d.writeFileNum, &d.writeMessages, &d.writePos)
 	} else {
 		_, err = fmt.Fscanf(f, "%d\n%d,%d\n%d,%d\n",
 			&d.depth,
```

Comment:
nit: d.depth -> depth, which would decrease the diffs between current request and upstream.
```go

	var depth int64
	// if user is using disk space limit feature
	if d.enableDiskLimitation {
		_, err = fmt.Fscanf(f, "%d\n%d,%d,%d\n%d,%d,%d\n",
			&depth,
			&d.readFileNum, &d.readMessages, &d.readPos,
			&d.writeFileNum, &d.writeMessages, &d.writePos)
	} else {
		_, err = fmt.Fscanf(f, "%d\n%d,%d\n%d,%d\n",
			&depth,
			&d.readFileNum, &d.readPos,
			&d.writeFileNum, &d.writePos)
	}

	if err != nil {
		return err
	}
	d.depth = depth
	d.nextReadFileNum = d.readFileNum
	d.nextReadPos = d.readPos
```

Reply:
Nice catch!